docs/api-documentation/index.html

```diff
@@ -51,8 +51,8 @@ <h1 class="govuk-heading-m">Releases</h1>
 <ul class="govuk-list govuk-list--bullet">
   <ul class="govuk-list govuk-list--bullet">
     <li>
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.7/">
-        Version 2.7 - Released to Sandbox on 15/10/2020. Released to Production on 22/10/2020.
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.8/">
+        Version 2.8 - Released to Sandbox on 29/10/2020. Due to release to Production on 02/11/2020.
```

Comment:
```suggestion
        Version 2.8 - Released to Sandbox on 29/10/2020. Due for release to Production on 02/11/2020.
```
Just to match the upcoming wording